api/v1alpha3/types.go

```diff
@@ -392,6 +269,7 @@ type AzureSharedGalleryImage struct {
 	Version string `json:"version"`
 }
 
+// AvailabilityZone - DEPRECATED: use FailureDomain instead
```

Comment:
nit: can we use the standard go lang deprecation notice 
```
"Deprecated:" followed by some information about the deprecation
```
source from here https://blog.golang.org/godoc

Reply:
Done, followed the examples in https://golang.org/search?q=Deprecated: